SCMI installer is missing the /bin/ path in prerequisite checks

The following occurs at instantiation using the command: 

`$ docker run   --rm   --privileged   --volume /:/media/root   jdeathe/centos-ssh:2.5.0   /usr/sbin/scmi install     --chroot=/media/root  -m systemd   --tag=2.5.0  --register   --name=ssh.3   --env='SSH_SUDO="ALL=(ALL) NOPASSWD:ALL"' --env='SSH_USER="centos"' --setopt="--volume {{NAME}}.config-ssh:/etc/ssh"`

`[ERROR] Missing required chroot command: systemctl`

Perhaps I'm misunderstanding what the use of `-m systemd` is, but at any rate, it still seems like the aforementioned error needs to be addressed.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,10 @@
 
 Summary of release changes for Version 2 - CentOS-7
 
+### 2.5.1 - Unreleased
+
+- Fixes `scmi` installation error when using the `--manager=systemd` option on Ubuntu hosts.
+
 ### 2.5.0 - 2019-01-28
 
 - Updates `openssl` package to 1.0.2k-16.el7.

--- a/src/usr/sbin/scmi
+++ b/src/usr/sbin/scmi
@@ -990,6 +990,7 @@ function scmi_manager_type_command_prerequisites ()
 	local -a COMMAND_PATHS=(
 		'/usr/local/bin'
 		'/usr/bin'
+		'/bin'
 	)
 	local MANAGER_TYPE="${1:-${SCMI_MANAGER_TYPE}}"
 

